features/lwipstack/lwipopts.h

```diff
@@ -331,6 +331,8 @@
 #define LWIP_ETHERNET               0
 #endif // MBED_CONF_LWIP_ETHERNET_ENABLED
 
+#define LWIP_L3IP   				1
```

Comment:
I think this needs similar configuration option as ethernet to mbed_lib.json. And should be disabled by default.

Reply:
changed